src/main/kotlin/endpoint/eth/EthNotaryResponse.kt

```diff
@@ -68,7 +65,7 @@ class EthNotaryResponseMoshiAdapter {
     @FromJson
     fun fromJson(layer: EthNotaryResponseLayer) = when (layer.type) {
         EthNotaryResponseType.Successful -> EthNotaryResponse.Successful(layer.ethSignature!!, layer.ethRefund!!)
-        EthNotaryResponseType.Error -> EthNotaryResponse.Error(layer.code!!, layer.reason!!)
+        EthNotaryResponseType.Error -> EthNotaryResponse.Error(layer.reason!!)
```

Comment:
Are you sure reason is always not null?

Reply:
Yes, it is for boilerplate code for serialisation.